collections/_article/dev-snapshot-godot-4-2-beta-5.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Dev snapshot: Godot 4.2 beta 5"
+excerpt: "More critical fixes are coming your way on the road to Godot 4.2 stable, including smoother transition for 3D projects made in 4.1."
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-2-beta-5.webp
+image_caption_title: Liberation
+image_caption_description: A game by Luke Miller
+date: 2023-11-07 12:00:00
+---
+
+Hope you've had a great weekend! We certainly enjoyed hosting a conference for our community — [GodotCon 2023](https://conference.godotengine.org/), with two dozen talks and workshops, giving insights into using the engine, and building it as well. The talks are available online over at our partners at [Chaos Computer Club](https://media.ccc.de/c/godotcon2023), and soon will be uploaded to the [official YouTube channel](https://www.youtube.com/@GodotEngineOfficial/videos) as well.
```

Comment:
Is "partners" best? It sounds a bit like a business relationship (maybe it is!). If not, maybe "friends"?